frontend-svelte/src/components/widgets/buttons/TextButton.svelte

```diff
@@ -38,6 +38,12 @@
 
 <style lang="scss" global>
 	.text-button {
+		span {
+			overflow: hidden;
+			text-overflow: ellipsis;
+		}
```

Comment:
Move down to be just above the `.icon-label {` line (with one blank line above and below as standard).